lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -475,6 +514,51 @@ ExhaustiveSynthesis::synthesize(SMTLIBSolver *SMTSolver,
     llvm::errs() << "There are " << Guesses.size() << " Guesses\n";
 
 
+  std::map<Inst *, Inst *> InstCache;
+  std::map<Block *, Block *> BlockCache;
+
+  if (UseAlive) {
+    std::set<const Inst *> Visited;
+    if (exceeds64Bits(LHS, Visited))
+      llvm::report_fatal_error("LHS exceeds 64 bits");
+
+    Inst *Ante = IC.getConst(APInt(1, true));
+    for (auto PC : PCs ) {
+      Inst *Eq = IC.getInst(Inst::Eq, 1, {PC.LHS, PC.RHS});
+      Ante = IC.getInst(Inst::And, 1, {Ante, Eq});
+    }
+
+    AliveDriver Verifier(LHS, Ante);
+    for (auto &&G : Guesses) {
+      std::set<const Inst *> Visited;
+      auto C = findConst(G, "reservedconst_0", Visited);
+      if (!C) {
+        if (Verifier.verify(G)) {
+          RHS = G;
+          return EC;
+        }
+      } else {
+        auto Result = Verifier.synthesizeConstant(G);
+        // TODO(manasij7479): Extend to multiple constants
+        // TODO: Counterexample guided loop or UB constraints in query
+        if (Result.has_value()) {
+          if (C) {
```

Comment:
execution would not reach this line unless C were true, so this test needs to go away